From 90b0130deec154dfd415c8d6d422c6e6140f8bf1 Mon Sep 17 00:00:00 2001
From: Todd Lipcon <todd@apache.org>
Date: Tue, 10 Apr 2012 22:31:18 +0000
Subject: [PATCH 0101/1357] HDFS-3243. TestParallelRead timing out on jenkins. Contributed by Henry Robinson.

git-svn-id: https://svn.apache.org/repos/asf/hadoop/common/trunk@1312029 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit 152ca47a787c969c4ffb3c96ebca0be4da76bbc8)
---
 .../apache/hadoop/hdfs/TestParallelReadUtil.java   |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestParallelReadUtil.java b/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestParallelReadUtil.java
index 0a0fc67..8e218d6 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestParallelReadUtil.java
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestParallelReadUtil.java
@@ -179,7 +179,7 @@ public class TestParallelReadUtil {
    */
   static class ReadWorker extends Thread {
 
-    static public final int N_ITERATIONS = 1024 * 4;
+    static public final int N_ITERATIONS = 1024;
 
     private static final double PROPORTION_NON_POSITIONAL_READ = 0.10;
 
-- 
1.7.0.4

